configs/llama-v2-7b/README.md

```diff
@@ -42,40 +42,40 @@ To use `prompt flow` in VS Code, please refer to this [Quick Start](https://micr
 2. Make sure you have a [HuggingFace account](https://huggingface.co/) and [generate an access token](https://huggingface.co/docs/hub/security-tokens)
 3. Accept the LICENSE of [llama](https://huggingface.co/meta-llama/Llama-2-7b-hf) on HuggingFace. 
 
 ### Provision Azure Resources
-To get started, you need provision the Azure Resource for remote fine-tuning. This can be done by running the `AI Toolkit: Provision Azure Container Apps job for fine-tuning.` from command palette.
+To get started, you need to provision the Azure Resource for remote fine-tuning. Do this by running the `AI Toolkit: Provision Azure Container Apps job for fine-tuning` from the command palette.
 
-You can monitor the progress of the provision via the link that is displayed in the output channel.
+Monitor the progress of the provision through the link displayed in the output channel.
 
-### Add Huggingface Token to the ACA Secret
+### Add Huggingface Token to the Azure Container App Secret
 If you're using Llama, ensure to accept the LICENSE of [llama](https://huggingface.co/meta-llama/Llama-2-7b-hf) on HuggingFace. 
-Following this, to bypass the need for manual login on the Hugging Face Hub, you should set your HuggingFace token as an environment variable. 
+Following this, set your HuggingFace token as an environment variable to avoid the need for manual login on the Hugging Face Hub.
 You can do this using the `AI Toolkit: Add Azure Container Apps Job secret for fine-tuning command`. With this command, you can set the secret name as [`HUGGING_FACE_HUB_TOKEN`](https://huggingface.co/docs/huggingface_hub/package_reference/environment_variables#hftoken) and use your Hugging Face token as the secret value.
 
 ### Run Fine-tuning
 To start the remote fine-tuning job, execute the `AI Toolkit: Run fine-tuning` command.
 
 To view the system and console logs, you can visit the Azure portal using the link in the output panel (more steps at [View and Query Logs on Azure](https://aka.ms/ai-toolkit/remote-provision#view-and-query-logs-on-azure)). Alternatively, you can view the console logs directly in the VSCode output panel by running the command `AI Toolkit: Show the running fine-tuning job streaming logs`. 
-> **Note:** The job might take a few minutes to initiate. If there is already a running job, the current one may be queued to start later.
-    
+> **Note:** The job might be queued due to insufficient resources. If the log is not displayed after execute the `AI Toolkit: Show the running fine-tuning job streaming logs` command, wait for a while and then execute the command again to re-connect to the streaming log.
```

Comment:
Add comma after 'displayed' and remove 'after', i.e. If the log is not displayed, execute the...

Reply:
Updated